src/main/java/core/basesyntax/Colors.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax;
+
+public enum Colors {
```

Comment:
As enum contains a set of constants and according to Java naming convention a “constant” must be in uppercase, therefore, all the values within enum should be written in Capital letters.